doc/debug.md

```diff
@@ -13,8 +13,7 @@ class MyGame extends Game with FPS {
 
 ## BaseGame features
 
-Flame provides some features for debugging, these features are enabled when the method `debugMode` from the `BaseGame` class is overridden, and returning `true`. When it's enabled all `PositionComponent`s will be wrapped into a rectangle, and have its position rendered on the screen, so you can visually verify the component boundaries and position.
-
-In addition to the debugMode, you can also ask `BaseGame` to record the fps(the `BaseGame` used the [FPSCounter](fps-counter) mixin). To enable it you have to override the `recordFps` method to return `true`, by doing so, you can access the current fps by using the method `fps`.
```

Comment:
Why remove this? We still have the FPSCounter feature right?

Reply:
The information about FPSCounter is just a few lines up, in the same doc.

Reply:
Fair enough